server/data/mappers/educationMapper.ts

```diff
@@ -0,0 +1,8 @@
+import type { EducationResponse } from 'educationAndWorkPlanApiClient'
+import type { EducationDto } from 'inductionDto'
+
+function toCreateEducationRequest(educationDto: EducationDto): EducationResponse {
```

Comment:
Ah ha 😁 , I got confused about this when reading the test above ☝️ 

At first I was about to say "no, we don't want to create a EducationRequest object (yet, we will do in a later story)"

But then I saw the code here and realised it was a copy and paste typo 🤣 

I think the function should be called `toEducationResponse` , and therefore change "request" to "response" in your test class above ☝️ 
👍

Reply:
Refactored.